src/IssueViz/IssueVisualizationControl/IssueVisualizationControl.xaml

```diff
@@ -125,10 +125,16 @@
                     <Setter Property="Background" Value="{DynamicResource {x:Static vsShell:VsBrushes.CommandBarHoverOverSelectedKey}}"/>
                     <Setter Property="BorderBrush" Value="{DynamicResource {x:Static vsShell:VsBrushes.CommandBarHoverOverSelectedIconBorderKey}}" />
                 </Trigger>
-                <Trigger Property="IsSelected" Value="True">
-                    <Setter Property="Background" Value="{DynamicResource {x:Static vsShell:VsBrushes.StartPageSelectedItemBackgroundKey}}"/>
-                    <Setter Property="BorderBrush" Value="{DynamicResource {x:Static vsShell:VsBrushes.StartPageSelectedItemBackgroundKey}}" />
-                </Trigger>
+                <MultiDataTrigger>
```

Comment:
This is a small UI hack.
Reminder of the binding cycle: the user click on an item in the list --> selection service is updated --> selection services raises an event --> UI is updated.
Unfortunately, if the item is not actually navigable, there is no way for me to break this cycle. I can't undo the user's selection, it's already too late. WPF does not have a BeforeSelectionChanged event and any trick that I tried (e.g. not setting the view model property to the new `value`, or not raising the selection event, or reverting it back to the previous selection, etc) has resulted in side effects.

So instead, I just make the item appear as not selected. 🤷‍♀️

Reply:
A simpler UX would just to allow all list items to be selected, and if the location is navigable then we open the file in the editor. The keyboard navigation commands could do the same. The behaviour would be consistent and the code simpler.

I'm not convinced that the UX of "only navigable items can be selected" is clearly more intuitive that "all items are selectable - navigate to the file if possible". What do we do in the other IDEs?

Reply:
As we discussed offline with the PM, lets keep the non-navigable items not selectable.